src/Rokt-Kit.js

```diff
@@ -413,6 +418,13 @@ var constructor = function () {
     function isKitReady() {
         return !!(self.isInitialized && self.launcher);
     }
+
+    function isPartnerInLocalLauncherTestGroup() {
+        return (
+            window.mParticle.config &&
+            window.mParticle.config.isLocalLauncherEnabled
```

Comment:
I know the server returns the string '"True"` and not the boolean true, for a lot of kit configurations.  Is that the case for general configs also?  If so, we'd want this to be 
```suggestion
            window.mParticle.config.isLocalLauncherEnabled === 'True'
```